redis/sentinel.py

```diff
@@ -240,7 +240,7 @@ def __init__(
         self.sentinel_kwargs = sentinel_kwargs
 
         self.sentinels = [
-            Redis(hostname, port, **self.sentinel_kwargs)
```

Comment:
As this is the only place where `self.sentinel_kwargs` are used we have to remove it